pkg/manifests/manifests_test.go

```diff
@@ -94,6 +109,7 @@ func TestIngressControllerResources(t *testing.T) {
 		require.NoError(t, err)
 
 		fixture := path.Join("fixtures", tc.Name) + ".json"
+		t.Logf("FILENAME: %s", tc.Name)
```

Comment:
Necessary?

Reply:
yeahhh it's otherwise very difficult to figure out which file the test failure takes place in